LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -23,9 +25,13 @@ public class TherapySettingsViewModel: ObservableObject {
     let syncPumpSchedule: PumpManager.SyncSchedule?
     let sensitivityOverridesEnabled: Bool
     let prescription: Prescription?
+    let appName: String
```

Comment:
We should probably get this via the environment before it spreads too far. @rickpasetto I think you wrote the original extension in Loop to do this, want to pull that down to LoopKit? Since it uses `Bundle.main`, I'd expect it work from within LoopKit too.

Reply:
@mpangburn I wasn't sure that was ok to do.  I'd be happy to do it...I'll put up a separate PR and add Pete to debate it :).